backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc

```diff
@@ -0,0 +1,30 @@
+[id="installing-oadp-mcg"]
+= Installing the Openshift API for Data Protection with Google Cloud Platform
```

Comment:
I am not sure if the title is correct. @apinnick could you please double confirm?

Reply:
Will check.